service/plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/constants/RouteConstants.java

```diff
@@ -9,7 +9,9 @@ public class RouteConstants {
 
     public static final String SCAN_ID = "SCAN_ID";
 
-    public static final String TRACE_ID = "TRACE_ID";
+    public static final String TRACE_ID = "traceId";
```

Comment:
Changed "TRACE_ID" to "traceId" so that the logger will automatically pick it up when using this as the MDC context key.